### DISCUSSION TITLE
Are you seeing these posts?

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/FIMS#239] [Polls]
Hi all, please reply in this poll what you think of using discussions.

**Comment by iantaylor-NOAA** [comment]
0 comments (before this one), but you did get 5 votes.

**Comment by nathanvaughan-NOAA** [comment]
Sorry Christine :( I guess 5 of us were onboard and for the rest Github discussions disappear into the email black hole. Is it bad that I think discussions are good even after I missed this a year ago :( I got Ian's response so I guess no excuse that I don't get the emails.

**Comment by kellijohnson-NOAA** [comment]
We have decided to continue to use GitHub Discussions rather than switching to a different platform like Slack or Discord. So, I am going to close this Discussion.
